oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexWriter.java

```diff
@@ -97,29 +98,30 @@ public boolean close(long timestamp) {
         return bulkProcessorHandler.close();
     }
 
-    protected void provisionIndex() throws IOException {
+    protected void provisionIndex(long seed) throws IOException {
         // check if index already exists
+        final String indexName = ElasticIndexNameHelper.getElasticSafeIndexName(
+                indexDefinition.getRemoteIndexAlias() + "-" + seed);
```

Comment:
Could we use Long.toHexString(seed)? It is a bit shorter (at most 16 characters instead of 20 for decimal)